api/src/Entity/ContentNode/Storyboard.php

```diff
@@ -81,20 +83,17 @@ public function removeSection(StoryboardSection $section): self {
 
     /**
      * @param Storyboard $prototype
+     * @param EntityMap  $entityMap
      */
-    public function copyFromPrototype($prototype) {
+    public function copyFromPrototype($prototype, &$entityMap = null) {
+        parent::copyFromPrototype($prototype, $entityMap);
+
         // copy all storyboard sections
-        foreach ($prototype->sections as $prototypeSection) {
+        foreach ($prototype->sections as $sectionPrototype) {
             $section = new StoryboardSection();
-
-            $section->column1 = $prototypeSection->column1;
-            $section->column2 = $prototypeSection->column2;
-            $section->column3 = $prototypeSection->column3;
-            $section->setPos($prototypeSection->getPos());
-
             $this->addSection($section);
-        }
 
-        parent::copyFromPrototype($prototype);
+            $this->copyFromPrototype($sectionPrototype, $entityMap);
```

Comment:
I think this should be `$section->copyFromPrototype($sectionPrototype, $entityMap)`